ts/src/core/db.ts

```diff
@@ -441,15 +440,15 @@ export class Postgres implements Connection {
     values?: any[],
   ): Promise<QueryResult<QueryResultRow>> {
     const r = await this.pool.query(query, values);
-    return r;
+    return r as QueryResult<QueryResultRow>;
   }
 
   async queryAll(
     query: string,
     values?: any[],
   ): Promise<QueryResult<QueryResultRow>> {
     const r = await this.pool.query(query, values);
-    return r;
+    return r as QueryResult<QueryResultRow>;
```

Comment:
hmm, why did we need this series of changes?

Reply:
My build kept failing without these. Maybe I'm using a later TS version? Not sure, but I can revert them if you want.